### PR TITLE
Includes the lti gem's Context into the Application controller (#46).

### PR DESCRIPTION
**Pull request by bwatson78**
app/controllers/application_controller.rb: Includes the lti gem's Context module.

### CODE REVIEW

**Comment by bwatson78** [comment]
@alexBLR I also added `lti` as a provider in the `OmniAuth::Builder`.

File: config/initializers/omniauth.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
   if !Rails.env.development? && !Rails.env.test?
+    provider :lti, :oauth_credentials => {:test => 'secret'}
```
Comment (alexBLR) [inline config/initializers/omniauth.rb:3]: I think this line is not needed.  https://github.com/emory-libraries/avalon/blob/master/config/settings.yml#L74-L90

**Review by alexBLR — CHANGES_REQUESTED** [review]
I just left one comment.

File: config/initializers/omniauth.rb (hunk 1 above)
Comment (bwatson78) [reply config/initializers/omniauth.rb]: Removed.

**Pull request merged**